databases/week3/Homework_week3.sql

```diff
@@ -0,0 +1,249 @@
+  values ('Spring Roll Wrappers', 'Wraps', '1 Laurel Street', '2020-03-27 02:55:23', 5, 11.32, '2020-03-20');
+insert into Meal (title, description, location, meal_time, max_reservations, price, created_date) 
+  values ('Pecan Raisin - Tarts', 'Tarts', '149 Westridge Way', '2020-04-17 06:41:29', 7, 5.14, '2020-04-10');
+insert into Meal (title, description, location, meal_time, max_reservations, price, created_date) 
+  values ('Pasta', 'Lasagna Noodle', '78064 Sherman Drive', '2020-01-23 19:46:26', 12, 134.81, '2020-01-10');
+insert into Meal (title, description, location, meal_time, max_reservations, price, created_date) 
+  values ('Lamb', 'Whole Head Off', '75827 Warner Alley', '2020-04-27 13:55:57', 25, 152.38, '2020-04-15');
+insert into Meal (title, description, location, meal_time, max_reservations, price, created_date) 
+  values ('Bread-Pita', 'Pita', '51934 Melrose Street', '2020-03-09 15:05:49', 15, 4.71, '2020-03-02');
+
+  
+-- Reservation
+insert into Reservation (number_of_guests,meal_id,created_date)
+ values(12, 7, '2020-03-20');
+insert into Reservation (number_of_guests,meal_id,created_date)
+ values(10, 11, '2020-01-22');
+insert into Reservation (number_of_guests,meal_id,created_date)
+ values(8, 8, '2020-02-02');
+insert into Reservation (number_of_guests,meal_id,created_date)
+ values(25, 12, '2020-04-25');
+insert into Reservation (number_of_guests,meal_id,created_date)
+ values(5, 10, '2020-04-15');
+insert into Reservation (number_of_guests,meal_id,created_date)
+ values(4, 9, '2020-03-26');
+ insert into Reservation (number_of_guests,meal_id,created_date)
+ values(12, 13, '2020-03-07');
+ 
+-- Review
+insert into Review (title, description, meal_id, stars,created_date)
+ values ('Chinese Food review', 'Excellent', 7, 5,'2020-03-22');
+insert into Review (title, description, meal_id, stars,created_date)
+ values ('Lamb review', 'Excellent', 12, 5,'2020-04-25');
+insert into Review (title, description, meal_id, stars,created_date)
+ values ('Pasta', 'Good', 11, 4,'2020-01-24');
+insert into Review (title, description, meal_id, stars,created_date)
+ values ('Spring Roll Wrapps review', 'Bad', 9, 1,'2020-03-28');
+insert into Review (title, description, meal_id, stars,created_date)
+ values ('White Baguette review', 'Ok', 8, 3,'2020-02-03');
+insert into Review (title, description, meal_id, stars,created_date)
+ values ('Pecan Raisin review', 'Very Good', 10, 4,'2020-04-15');
+insert into Review (title, description, meal_id, stars,created_date)
+ values ('Pita review', 'Good', 13, 4,'2020-03-15');
+insert into Review (title, description, meal_id, stars,created_date)
+ values ('Chinese Food review', 'Good', 7,4 ,'2020-03-22');
+ 
+-- QUERIES
+-- 1)Get meals that has a price smaller than a specific price fx 90
+SELECT * 
+FROM meal
+WHERE price < 90;
+
+-- 2)Get meals that still has available reservations
+SELECT* 
+FROM Meal m
+JOIN Reservation rs
+ON m.id = rs.meal_id
+WHERE max_reservations > number_of_guests;
+
```

Comment:
Please correct the above query:
```

SELECT*
FROM   meal m
       JOIN reservation rs
         ON m.id = rs.meal_id
WHERE  max_reservations > number_of_guests
GROUP  BY rs.meal_id
HAVING Sum(rs.number_of_guests) < m.max_reservations; 

```

Reply:
Hi Babak..
I have implemented the feedback but I am getting an error on this query:

    SELECT*
FROM   meal m
       JOIN reservation rs
         ON m.id = rs.meal_id
WHERE  max_reservations > number_of_guests
GROUP  BY rs.meal_id
HAVING Sum(rs.number_of_guests) < m.max_reservations;

Reply:
Hello, sorry for the late answer. The mails have strangely arrive pretty late.

Would you please tell me what error you receive. I have no errors.
I have ran your SQLs and then the correction I suggested and it shows no errors.
Thank you